specification/databricks/resource-manager/Microsoft.Databricks/preview/2025-08-01-preview/examples/EnableEncryption.json

```diff
@@ -0,0 +1,180 @@
+{
+  "parameters": {
+    "subscriptionId": "11111111-1111-1111-1111-111111111111",
+    "resourceGroupName": "rg",
+    "workspaceName": "myWorkspace",
+    "api-version": "2025-08-01-preview",
+    "parameters": {
+      "properties": {
+        "managedResourceGroupId": "/subscriptions/11111111-1111-1111-1111-111111111111/resourceGroups/myManagedRG",
+        "parameters": {
+          "customPrivateSubnetName": {
+            "type": "String",
+            "value": "PrivateBob"
+          },
+          "customPublicSubnetName": {
+            "type": "String",
+            "value": "PublicSarah"
+          },
+          "customVirtualNetworkId": {
+            "type": "String",
+            "value": "/subscriptions/11111111-1111-1111-1111-111111111111/resourceGroups/rg/providers/Microsoft.Network/virtualNetworks/myNetwork"
+          },
+          "prepareEncryption": {
+            "value": true
+          },
```

Comment:
Not sure I understand what the difference is between the EnableEncryption and PrepareEncryption examples? They look very similar.